src/SwaggerProvider.DesignTime/v3/OperationCompiler.fs

```diff
@@ -156,11 +156,7 @@ type OperationCompiler(schema: OpenApiDocument, defCompiler: DefinitionCompiler,
         let retMimeAndTy =
             let okResponse =
                 operation.Responses
-                |> Seq.tryFind(fun resp -> 
-                    resp.Key = "200" 
-                    || resp.Key.StartsWith("20")
-                    || resp.Key = "default"
-                )
+                |> Seq.tryFind(fun resp -> resp.Key = "200" || resp.Key.StartsWith("20") || resp.Key = "default")
```

Comment:
ah thanks, i tried to run fantomas format from vscode in my github codespace but was erroring and i didn't get what was wrong